fs/span-manager/span_manager_test.go

```diff
@@ -314,6 +322,10 @@ func TestValidateState(t *testing.T) {
 	}
 }
 
+// func TestGetSpanContent(t *testing.T) {
```

Comment:
Why the commented function?  Is this still to be added?

Reply:
I will move this out...